src/components/footer.tsx

```diff
@@ -53,6 +53,7 @@ export default function Footer() {
               <Link
                 href={item.href}
                 className="text-sm leading-6 text-gray-600 hover:text-gray-900 dark:text-gray-400 dark:hover:text-gray-500"
+                target={item.name === 'Discord' ? '_blank' : undefined}
```

Comment:
Might change undefined to a different value

Reply:
Yes changing it to _self which is default.